src/patternfly/components/Login/login.scss

```diff
@@ -226,6 +236,25 @@
   }
 }
 
+.pf-c-login__main-footer-grid {
+  display: grid;
+  grid-gap: var(--pf-c-login__main-footer-grid--GridGap);
+  grid-template-columns: var(--pf-c-login__main-footer-grid--GridTemplateColumns);
+  justify-items: left;
```

Comment:
Is this necessary, since `justify-items: center;` only applies in a small viewport?